test/onnx/gen_onnx.py

```diff
@@ -902,6 +902,49 @@ def initializer_not_an_input():
     return ([node], [x], [y], [w])
 
 
+@onnx_test
+def instance_norm_test():
+    x = helper.make_tensor_value_info('0', TensorProto.FLOAT, [1, 1, 3, 3])
```

Comment:
I would suggest changing the C in the shape to a number other than 1 to cover the scenario I mentioned above.

Reply:
fixed